### PR TITLE
General enhancements

### PR DESCRIPTION
**Pull request by lieutdan13**
WIP

### CODE REVIEW

**Comment by lieutdan13** [comment]
@karec Since you mentioned in #23 that you don't want to support Python 3.8, I will remove that from here. Otherwise, let me know if you agree/disagree with any of my commits. They are logically separated, so they should be easy to exclude.

**Comment by karec** [comment]
@lieutdan13 seems good to me, thanks again for this PR !

For python 3.8, sorry I wasn't clear, the point was that python 3.8 is stable and available now and from what I've seen, ubuntu LTS is on python 3.6 by default and debian buster come with python 3.7, so supporting 3.5 is clearly not a strict requirement for this project.

I'm merging this

**Comment by lieutdan13** [comment]
@karec fixed. I've removed python 3.5 and 3.8 from the project.

**Comment by karec** [comment]
@lieutdan13 thanks, tell me when you want me to start the review

**Comment by lieutdan13** [comment]
I am done for now. I'll be using it in its current state for my Hackathon tomorrow.

**Comment by karec** [comment]
Ok, thanks again for this PR I will review it ASAP and good luck for your Hackathon

File: {{cookiecutter.project_name}}/cookiecutter-options.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+default_context:
+    project_name: "{{ cookiecutter.project_name }}"
+    app_name: "{{ cookiecutter.app_name }}"
+    python_version: "{{ cookiecutter.python_version }}"
+    admin_user_username: "admin"
+    admin_user_email: "admin@brightedge.com"
```
Comment (karec) [inline {{cookiecutter.project_name}}/cookiecutter-options.yml]: set to `admin@mail.com` in `cookiecutter.json`

Comment (lieutdan13) [reply {{cookiecutter.project_name}}/cookiecutter-options.yml]: haha. Whoops!!

**Comment by karec** [comment]
One last thing: do you think you can update the README to include makefile usage ? If not I can do it later

**Comment by lieutdan13** [comment]
@karec Makefile docs added

File: README.md [hunk 2]
```diff
@@ -155,7 +161,9 @@ gunicorn myapi.wsgi:app
 
 And that's it ! Gunicorn is running on port 8000
 
-### Running with uwsgi
+If you chose gunicorn as your wsgi server, the proper commands should be in your dokcer-compose file.
```
Comment (karec) [inline README.md]: small typo `dokcer` instead of `docker`

File: README.md [hunk 3]
```diff
@@ -166,6 +174,7 @@ uwsgi --http 127.0.0.1:5000 --module myapi.wsgi:app
 
 And that's it ! Uwsgi is running on port 5000
 
+If you chose uwsgi as your wsgi server, the proper commands should be in your dokcer-compose file.
```
Comment (karec) [inline README.md]: small typo `dokcer` instead of `docker`

**Comment by karec** [comment]
Merged ! thanks again for this PR

**Pull request merged**